addons/website_slides/tests/test_attendee.py

```diff
@@ -441,10 +441,12 @@ def test_invite_route_errors_handling(self):
         self.channel.visibility = 'members'
 
         # No such channel
-        invite_url = "/slides/-1"
-        res = self.url_open(invite_url)
-        self.assertEqual(res.status_code, 200)
-        self.assertTrue('/slides?invite_error=no_channel' in res.url, "This channel does not exist. Redirect to the main /slides page.")
+        channel_id = -123
```

Comment:
Would instead rework this test case to use "negative-id" on self.channel and make sure it works.

In addition, would pick up the max existing ID (search sorted by id desc limit 1), and add 1 to that number, to make sure it redirects to "no_channel" in case we have something that does not exist.

That way we test everything.